cmd/nerdctl/network_prune.go

```diff
@@ -97,7 +99,14 @@ func networkPrune(cmd *cobra.Command, client *containerd.Client, ctx context.Con
 
 	var removedNetworks []string // nolint: prealloc
 	for _, net := range e.Networks {
-		if net.Name == "host" || net.Name == "none" {
+		keepNetwork := false
+		for _, d := range networkDriversToKeep {
+			if net.Name == d {
+				keepNetwork = true
+				continue
```

Comment:
Break

Reply:
right:) anyway you can switch this whole thing to `slices.Contains()` when changing go.mod to 1.18